app/assets/javascripts/services/project_roles_svc.js

```diff
@@ -0,0 +1,28 @@
+samson.service('projectRolesService', function ($http) {
+
+    this.loadProjectRoles = function () {
+        return $http.get('/project_roles');
+    };
+
+    this.createProjectRole = function (project_role) {
+        var data = {
+            project_role: {
+                user_id: project_role.user_id,
+                project_id: project_role.project_id,
+                role_id: project_role.role_id
+            }
+        };
```

Comment:
why not just
```
var data = {
  project_role: project_role
};
```
Sending the local uniqueID doesn't seem too bad a thing for more readable code.